.husky/pre-commit

```diff
@@ -2,4 +2,3 @@
 . "$(dirname "$0")/_/husky.sh"
 
 yarn build
```

Comment:
Since we aren't committing `/lib` anymore we shouldn't need this. We can remove the hook all together.